push_fix.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+export GIT_PAGER=cat
+export PAGER=cat
+cd /home/daniel/sure
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)